addons/account/views/account_bank_statement_views.xml

```diff
@@ -203,25 +203,26 @@
                                     <!-- Visible fields -->
                                     <field name="sequence" widget="handle"/>
                                     <field name="date"
-                                           attrs="{'readonly': [('parent.state', '!=', 'open')]}"/>
-                                    <field name="payment_ref"/>
+                                           attrs="{'readonly': ['|', ('is_reconciled', '=', True), ('parent.state', '!=', 'open')]}"/>
+                                    <field name="payment_ref"
```

Comment:
Does it make sense to have other fields depending on `('parent.state', '!=', 'open')`? Since `<field name="line_ids" attrs="{'readonly': [('state', '!=', 'open')]}"`...